internal/pkg/docker/dockercompose/unsupported.go

```diff
@@ -3,17 +3,15 @@
 
 package dockercompose
 
-// NOTE: This file is currently unused, it will be used in the next PR. Don't mind the comments for now!
-
 // IgnoredKeys stores a list of keys in the Compose YAML that couldn't be processed,
 // but are likely to not be significant enough to cause the converted application to
 // fail. It's expected that this will eventually be displayed to the user.
 type IgnoredKeys []string
 
-/*
 // ignoredServiceKeys lists out the keys on Compose services that are ignored in conversion.
 //
 // note: build keys are handled separately in convertBuildConfig
+// note: using a map as a set here
```

Comment:
this is a fairly common use case, probably don't need a comment for it
```suggestion
```

i like the way `map[string]bool` reads more - ie `if ignoredServiceKey["key"]` vs `if _, ok := ignoredServiceKey["key"]; ok `, but whichever you prefer is fine